share/wake/lib/system/job.wake

```diff
@@ -91,6 +91,7 @@ tuple Plan =
   export Usage:        Usage        # User-supplied usage prediction; overruled by database statistics (if any)
   export FnInputs:     (List String => List String) # Modify the Runner's reported inputs  (files read)
   export FnOutputs:    (List String => List String) # Modify the Runner's reported outputs (files created)
+  export Label:        String       # Human-readable label chosen by Plan author to describe the job intent
```

Comment:
```suggestion
  export Label:        String       # Machine-readable label chosen by Plan author to describe the job intent
```

I feel that "human-readable" might be misleading here, since it's actually being used to programmatically filter for specific plans. It's less important that it is human-readable and more important that it is a machine-compariable label.